ui/selectors/selectors.js

```diff
@@ -2977,8 +2978,22 @@ export function getMetaMetricsDataDeletionStatus(state) {
   return state.metamask.metaMetricsDataDeletionStatus;
 }
 
+/**
+ * Gets the remote feature flags by combining flags from both the manifest and state.
+ * Manifest flags take precedence and will override any duplicate flags from state.
+ * This allows for both static (manifest) and dynamic (state) feature flag configuration.
+ *
+ * @param {object} state - The MetaMask state object
+ * @returns {object} Combined feature flags object with manifest flags taking precedence over state flags
+ */
 export function getRemoteFeatureFlags(state) {
```

Comment:
Can we please put this selector in a different file besides `selectors.js`?  We should really be making this file shorter, it's important to circular dependencies, React.lazy, and performance.

Reply:
Actually David proposed the same solution, now it makes sense to remove it out of the large selector, refactored in 5308bed8ec062195685cd4199d01f306ff422009 and 2cc15607d22f8ff797ff7827b7bc9e02564aafa4